### PR TITLE
Chaplin: Update to NPM, Fix bugs

### PR DESCRIPTION
**Pull request by yavorsky**
`vendor.js`: for avoiding `Uncaught Error: Chaplin requires Common.js or AMD modules`.
Brunch doesn't handle node_modules yet, so it's impossible to refer to node_modules files from `public/` dir.

### CODE REVIEW

File: examples/chaplin-brunch/.gitignore [hunk 1]
```diff
@@ -1,8 +1,28 @@
-bower_components/
-!bower_components/todomvc-common/
+# NPM packages.
```
Comment (arthurvr) [inline examples/chaplin-brunch/.gitignore]: This comment seems like not necessary to me. We all know what `node_modules` are right?

File: examples/chaplin-brunch/.gitignore [hunk 2]
```diff
@@ -1,8 +1,28 @@
...
+!node_modules/todomvc-app-css/index.css
+
+node_modules/todomvc-common/*
+!node_modules/todomvc-common/base.css
+!node_modules/todomvc-common/base.js
+
+node_modules/coffee-script-brunch/
+node_modules/javascript-brunch/
+node_modules/handlebars-brunch/
+node_modules/uglify-js-brunch/
```
Comment (arthurvr) [inline examples/chaplin-brunch/.gitignore]: If you want to ignore a whole folder, you should [have an asterisk](https://github.com/tastejs/todomvc/pull/1142).

**Comment by sindresorhus** [comment]
- [bug] When you mark a todo as complete the text doensn't get the strikethrough.
- [bug] Double-clicking the todo places the cursor at the start of the text, not the end.

**Comment by yavorsky** [comment]
Fixed

File: examples/chaplin-brunch/package.json [hunk 3]
```diff
@@ -1,30 +1,17 @@
...
-    "start": "brunch watch --server",
-    "test": "brunch test"
-  },
+  "private": true,
   "dependencies": {
-    "javascript-brunch": ">= 1.0 < 1.8",
-    "coffee-script-brunch": ">= 1.0 < 1.8",
+    "chaplin": "^1.0.0",
+    "exoskeleton": "^0.6.3",
+    "backbone.localstorage": "git://github.com/yavorsky/Backbone.localStorage.git#a47fe5aa410f70dae9066c439d84483e4b32507e",
```
Comment (sindresorhus) [inline examples/chaplin-brunch/package.json]: Can you use the official release here too?

Comment (yavorsky) [inline examples/chaplin-brunch/package.json]: @sindresorhus :+1:

**Comment by sindresorhus** [comment]
Thank you :)

**Pull request closed without merging**